vavr/generator/Generator.scala

```diff
@@ -1347,6 +1348,16 @@ def generateMainClasses(): Unit = {
           public static void println() {
               System.out.println();
           }
+
+          /**
+           * Using the {@code System.out.println()} will print all the objects passed as argument
+           * separating them by a space character (' ').
+           *
+           * @param objs The list of <code>Object</code> to be printed
```

Comment:
I would write

```
@param objs The objects to be printed
```

because it is no list. It could be 'array' or 'sequence' - but I think just 'objects' is enough